PDF_UA/2/8.2 Logical structure/8.2.5 Additional requirements for specific structure types/8.2.5.25 List (L, LI, LBody)/verapdf-profile-8-2-5-25-t01.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<profile xmlns="http://www.verapdf.org/ValidationProfile" flavour="PDFUA_2">
+    <details creator="veraPDF Consortium" created="2023-03-13T10:58:04.098+03:00">
+        <name>ISO 14289-2:202x - 8.2 logical structure - 8.2.5.25 List (L, LI, LBody) - ListNumbering attribute</name>
+        <description>If Lbl structure elements are present, the ListNumbering attribute shall be present on the 
+            respective L structure element; in such cases the value None shall not be used</description>
+    </details>
+    <hash></hash>
+    <rules>
+        <rule object="SEL">
+            <id specification="ISO_14289_2" clause="8.2.5.25" testNumber="1"/>
+            <description>If Lbl structure elements are present, the ListNumbering attribute shall be present on the
+                respective L structure element; in such cases the value None shall not be used</description>
+            <test>containsLabels == false || ListNumbering != 'None'</test>
+            <error>
+                <message>If Lbl structure elements are present, the ListNumbering attribute is not present on the
```

Comment:
List items contain Lbl structure elements, but the ListNumbering attribute is not present on the respective L structure element or it has value None